snapcraft/cli/containers.py

```diff
@@ -0,0 +1,58 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import click
+import os
+
+import snapcraft
+from snapcraft.internal import errors, lxd
+from ._options import get_project_options
+from . import env
+
+
+@click.group()
+def containerscli():
+    pass
+
+
+@containerscli.command()
+@click.option('--debug', is_flag=True,
+              help='Shells into the environment if the build fails.')
+def refresh(debug, **kwargs):
+    """Refresh an existing LXD container.
+
+    \b
+    Examples:
+        SNAPCRAFT_CONTAINER_BUILDS=1 snapcraft refresh
```

Comment:
Can you omit `SNAPCRAFT_CONTAINER_BUILDS` and instead make this command optionally show up by means of doing
```python
if not os.environ.get('SNAPCRAFT_CONTAINER_BUILDS'):
    command.pop(commands.index('refresh'))
```

Right after https://github.com/snapcore/snapcraft/blob/master/snapcraft/cli/__init__.py#L92

Reply:
Done. I used `use_container` here since eventually it won't be just the environment variable.